modules/deploy/pages/deployment-option/cloud/manage-billing/index.adoc

```diff
@@ -0,0 +1,4 @@
+= Manage Billing
+:description: Learn about Redpanda consumption metrics and subscriptions to Redpanda Cloud with discounted commitment billing.
```

Comment:
Learn about how Redpanda measures consumption through usage metrics in Redpanda Cloud.